server/backend/database/mongo/client.go

```diff
@@ -769,6 +769,9 @@ func (c *Client) FindDocInfosByKeys(
 	projectID types.ID,
 	docKeys []key.Key,
 ) ([]*database.DocInfo, error) {
+	if len(docKeys) == 0 {
+		return []*database.DocInfo{}, nil
```

Comment:
https://github.com/uber-go/guide/blob/master/style.md#nil-is-a-valid-slice

```suggestion
		return nil, nil
```